cmd/jaeger/internal/integration/es_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024 The Jaeger Authors.
```

Comment:
It looks like the CI is currently failing on the v2 tests, while v1 are good. I suggest splitting this PR. Let's first merge the change to v1 tests to use the new Purge function from the factory, and once it's confirmed working we can add v2 tests.